data/data/azure/vnet/internal-lb.tf

```diff
@@ -10,7 +10,7 @@ resource "azurerm_lb" "internal" {
 
   frontend_ip_configuration {
     name                          = local.internal_lb_frontend_ip_configuration_name
-    subnet_id                     = azurerm_subnet.master_subnet.id
+    subnet_id                     = var.master_subnet_id
```

Comment:
this should be the data_ source or the resource not the variable.

Reply:
I've modified this to be a local variable.

Reply:
shouldn't this be `local.worker_subnet_id` ??

Reply:
Oops. Fixed!